README.md

```diff
@@ -1 +1,26 @@
 # hw01
+
+## Biography of David Kanos
+
+I was born on *April 24th, 1996* in **Jos, Nigeria**  I am currently getting my Masters in Public Policy at **The University of Chicago.**
+
+### Family
+
+I have two younger sisteres named *Elsa* and *Natalia* and I love them both very much. My mum runs and NGO in Nigeria that helps widows and orphans affected by Boko Haram in Nigeria. My dad is an investor and consoultant in Nigeria. 
+
+#### My Hobbies
```

Comment:
I also agree with Yuwen that some images in addition to your text could have made the project more interesting (not necessary, though, since you did use four different types of R elements).